src/d3dx12_property_format_table.cpp

```diff
@@ -20,10 +20,10 @@
 #ifndef _WIN32
     #include <wsl/winadapter.h>
 #endif
-#include "d3dx12_property_format_table.h"
+#include <directx/d3dx12_property_format_table.h>
```

Comment:
`clang failed with stderr: C:\code\cpp\DirectX-Headers\src\d3dx12_property_format_table.cpp:23:10: fatal error: 'd3dx12_property_format_table.h' file not found`

I've no idea how this compiled for you guys, but that's definitely not right.

Reply:
These changes won't work in all the contexts these files are replicated, which is why we have this handled in the build setup rather than through the code.